python/ray/serve/controller.py

```diff
@@ -418,5 +435,43 @@ def list_deployments(
             )
         }
 
-    def get_deployment_statuses(self) -> Dict[str, DeploymentStatusInfo]:
-        return self.deployment_state_manager.get_deployment_statuses()
+    def list_deployments(self, include_deleted: Optional[bool] = False) -> bytes:
+        """Gets the current information about all deployments.
+
+        Args:
+            include_deleted(bool): Whether to include information about
+                deployments that have been deleted.
+
+        Returns:
+            DeploymentRoute's protobuf
```

Comment:
```suggestion
            DeploymentRouteList's protobuf serialized bytes
```

Reply:
It has been revised according to the suggested change.